### ISSUE TITLE
Allow "flattening" in tree view?

### ISSUE THREAD

**Issue by danqing** [atom/tree-view#615]
I didn't see an option or an issue related to this and I'm not sure if this has been raised before, but I wonder if tree-view can support flattening? Will be very useful for Java and the like.

```
/com
  /github
    /atom
      /...

-> /com/github/atom
     /...
```

**Comment by 50Wliu** [comment]
Hi @danqing, this seems like a duplicate of #125 and so will be fixed by #608 if it gets merged.

**Comment by danqing** [comment]
Sweet thanks! Sorry for not noticing that issue.
